bftclient/test/bft_client_api_tests.cpp

```diff
@@ -340,7 +342,8 @@ TEST_F(ClientApiTestFixture, primary_gets_learned_on_successful_write_and_cleare
       if (not_heard_from_yet.size() == 1) {
         quorum_of_replies_sent = true;
       }
-      client_receiver->onNewMessage((NodeNum)msg.destination.val, (const char*)reply.data(), reply.size());
+      client_receiver->onNewMessage(
+          (NodeNum)msg.destination.val, (const char*)reply.data(), reply.size(), (NodeNum)msg.destination.val);
```

Comment:
No need to pass the same value twice - let the default value take place.

Reply:
updated.